build from source

jjhelmus opened the pull request:
Build package from source using bazel
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

jjhelmus committed:
sync run requirements with upstream

jjhelmus committed:
build from source

jjhelmus committed:
add jjhelmus as maintainer

jjhelmus committed:
bump build number

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * `noarch` packages can't have selectors. If the selectors are necessary, please remove `noarch: python`.

jjhelmus commented:
@conda-forge-admin, please rerender

jjhelmus commented:
> * `noarch` packages can't have selectors. If the selectors are necessary, please remove `noarch: python`.

They can in this case, because things are complicated.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to cf_pr for you, but it looks like I wasn't able to push to the jjhelmus branch of tensorflow-hub-feedstock/re-render. Did you check the "Allow edits from maintainers" box?

jjhelmus committed:
MNT: Re-rendered with conda-build 3.18.9, conda-smithy 3.4.1, and conda-forge-pinning 2019.08.22

jjhelmus committed:
put lower bound on bazel version

This will end up pulling bazel from defaults until the conda-forge
package is updated.

jjhelmus committed:
ignore compiler run_exports

ocefpaf commented:
Thanks @jjhelmus! We should fix the linter and not the recipe.
